<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewPointDialog_Widget</class>
 <widget class="QWidget" name="ViewPointDialog_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CViewPointDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_layerName">
           <property name="text">
            <string>LayerName:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_longitude">
           <property name="text">
            <string>Longitude:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_latitude">
           <property name="text">
            <string>Latitude:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_height">
           <property name="text">
            <string>Height:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_pitch">
           <property name="text">
            <string>Pitch:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_heading">
           <property name="text">
            <string>Heading:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_flyTime">
           <property name="text">
            <string>FlyTime:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEdit_layerName"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QDoubleSpinBox" name="dSpinBox_longitude">
             <property name="decimals">
              <number>6</number>
             </property>
             <property name="minimum">
              <double>-180.000000000000000</double>
             </property>
             <property name="maximum">
              <double>180.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="maximumSize">
              <size>
               <width>37</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>째</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QDoubleSpinBox" name="dSpinBox_latitude">
             <property name="decimals">
              <number>6</number>
             </property>
             <property name="minimum">
              <double>-90.000000000000000</double>
             </property>
             <property name="maximum">
              <double>90.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="maximumSize">
              <size>
               <width>37</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>째</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QDoubleSpinBox" name="dSpinBox_height">
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-11000000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>1000000000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="maximumSize">
              <size>
               <width>37</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>m</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QDoubleSpinBox" name="dSpinBox_pitch">
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-180.000000000000000</double>
             </property>
             <property name="maximum">
              <double>180.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="maximumSize">
              <size>
               <width>37</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>째</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QDoubleSpinBox" name="dSpinBox_heading">
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-180.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="maximumSize">
              <size>
               <width>37</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>째</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QDoubleSpinBox" name="dSpinBox_flyTime">
             <property name="maximum">
              <double>99.989999999999995</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="maximumSize">
              <size>
               <width>37</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>s</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_currentView">
         <property name="text">
          <string>CurrentView</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resource/images.qrc"/>
 </resources>
 <connections/>
</ui>
